claim/README.md

```diff
@@ -293,6 +293,8 @@ You will still be able to see this node in your War Rooms in an **unreachable**
 If you want to reclaim this node into a different Space, you need to create a new identity by adding `-id=$(uuidgen)` to
 the claiming script parameters. For example, using the default claiming script:
```

Comment:
I would prefer if we added this sentence in the paragraph above, or below the code block.

```suggestion
the claiming script parameters. Make sure that you have the `uuidgen-runtime` packagen installed, as it is used to run the command `uuidgen` For example, using the default claiming script:
```

Reply:
done @joelhans

Reply:
Thanks! I requested reviews from a few engineers related to claiming.